zebra-network/src/peer_set/candidate_set.rs

```diff
@@ -109,21 +114,30 @@ pub(super) struct CandidateSet<S> {
     pub(super) address_book: Arc<std::sync::Mutex<AddressBook>>,
     pub(super) peer_service: S,
     next_peer_min_wait: Sleep,
+    next_update_time: Instant,
 }
 
 impl<S> CandidateSet<S>
 where
     S: Service<Request, Response = Response, Error = BoxError>,
     S::Future: Send + 'static,
 {
-    /// The minimum time between successive calls to `CandidateSet::next()`.
+    /// The minimum time between successive calls to [`CandidateSet::next()`][Self::next].
     ///
     /// ## Security
     ///
     /// Zebra resists distributed denial of service attacks by making sure that new peer connections
     /// are initiated at least `MIN_PEER_CONNECTION_INTERVAL` apart.
     const MIN_PEER_CONNECTION_INTERVAL: Duration = Duration::from_millis(100);
 
+    /// The minimum time between successive calls to [`CandidateSet::update()`][Self::update].
+    ///
+    /// ## Security
+    ///
+    /// Zebra resists distributed denial of service attacks by making sure that requests for more
+    /// peer addresses are sent to the same peer at least `MIN_PEER_GET_ADDR_INTERVAL` apart.
```

Comment:
Most requests are sent to random peers. But that's ok, because if we rate-limit requests to _any_ peers, we also rate limit requests to _the same_ peer.

```suggestion
    /// Zebra resists distributed denial of service attacks by making sure that requests for more
    /// peer addresses are sent at least `MIN_PEER_GET_ADDR_INTERVAL` apart.
```